packages/alchemy/src/signer/client/index.ts

```diff
@@ -513,6 +513,14 @@ export class AlchemySignerClient {
       signal: options?.signal,
     });
 
+    // on iOS sometimes this is returned as empty or null, so handling that here
+    if (attestation.transports == null || attestation.transports.length === 0) {
+      attestation.transports = [
+        "AUTHENTICATOR_TRANSPORT_INTERNAL",
```

Comment:
`import { TurnkeyClient, getWebAuthnAttestation } from "@turnkey/http";`

this is returning the attestation. is this their SDK issue for not handling ios?

Reply:
yea this is something they should handle imo, but I wanna unblock dubz

Reply:
@moldy530